xml.go

```diff
@@ -138,6 +166,7 @@ func WithEntryFormattedTime(formatted bool) EntryOption {
 type Entry struct {
 	UUID            UUID              `xml:"UUID"`
 	IconID          int64             `xml:"IconID"`
+	CustomIconUUID  UUID              `xml:"CustomIconUUID"`
```

Comment:
This field is not there when no custom icon is set and the XML is marshalled by original KDBX. Therefore this should be optional